Fitness

NoahE321 opened the pull request:
milestone 1

NoahE321 committed:
Added class

MaherHssn committed:
added classes

:)

NoahE321 committed:
Added Classes and renamed Main

NoahE321 committed:
Renamed

MaherHssn committed:
variables (sleep sched)

NoahE321 committed:
Added blank methods

B1gCD committed:
basic variables/setup

Focusing on theory and planning

NoahE321 committed:
Create GeneralInfo.java

B1gCD committed:
Merge branch 'Fitness' of https://github.com/Ye-ICS/fss-everything-app into Fitness

NoahE321 committed:
More blank methods

NoahE321 committed:
Merge branch 'main' into Fitness

NoahE321 committed:
added questionaire to main

NoahE321 committed:
added booleans

NoahE321 committed:
Added code to methods

MaherHssn committed:
First Commit

added fxmls and controller

MaherHssn committed:
Signing in fxml and controller added

NoahE321 committed:
Merge branch 'main' into Fitness

MaherHssn committed:
Merge branch 'main' into FitnessUI

NoahE321 committed:
cleaned up methods and rearranged

NoahE321 committed:
Merge branch 'finess-methods-1' into Fitness

B1gCD committed:
Basic code

boo

B1gCD committed:
Merge branch 'FitnessMethods2' into Fitness

NoahE321 committed:
Update WorkoutManager.java

MaherHssn committed:
Commit #2

B1gCD committed:
Basic code for sleep manager

NoahE321 committed:
Update WorkoutManager.java

MaherHssn committed:
merge commit

MaherHssn committed:
Merge branch 'FitnessUI' into Fitness

MaherHssn committed:
Merge branch 'Fitness' of https://github.com/Ye-ICS/fss-everything-app into Fitness

B1gCD committed:
Merging

B1gCD committed:
Updated

B1gCD committed:
Sleep schedule update

MaherHssn committed:
added stuff

MaherHssn committed:
coomit

MrYe070 reviewed (changes requested):
Does not compile!

MaherHssn committed:
moved Controllers to Seperate file

MaherHssn committed:
m

MaherHssn committed:
View -> borderpane

changed all the fxml files which were view instead of borderpane so now it actually works :)

MaherHssn committed:
slashed out method :p

MaherHssn committed:
reworked signed in menu and finished signing in menu

MaherHssn committed:
added preference fxml and made it so you cant proceed if gender is not selected

MaherHssn committed:
preferences

NoahE321 committed:
Update Workout.java

NoahE321 committed:
Merge remote-tracking branch 'origin/FitnessUI' into finess-methods-1

NoahE321 committed:
Merge branch 'finess-methods-1' into Fitness

MaherHssn committed:
Merge remote-tracking branch 'origin/Fitness' into FitnessUI

B1gCD committed:
Bare minimum UI ready

B1gCD committed:
Merge branch 'FitnessMethods2' into Fitness

B1gCD committed:
Update SleepManager.java

B1gCD committed:
Merge branch 'FitnessMethods2' into Fitness

B1gCD committed:
Update SleepManager.class

NoahE321 committed:
Updated code for diet manager

NoahE321 committed:
Merge branch 'finess-methods-1' into Fitness

MaherHssn committed:
ADDED MORE MENUS!!!!!

MaherHssn committed:
Merge branch 'FitnessUI' into Fitness

NoahE321 committed:
Updated methods

NoahE321 committed:
Merge branch 'Fitness' into finess-methods-1

B1gCD committed:
Updated variables

B1gCD committed:
Merge branch 'Fitness' into FitnessMethods2

MaherHssn committed:
logmenu

MaherHssn committed:
made working bars for stuff

:))))))

NoahE321 committed:
Update DietManager.java

NoahE321 committed:
Update Diet.java

NoahE321 committed:
Merge branch 'finess-methods-1' into Fitness

MaherHssn committed:
hi!!

MaherHssn committed:
Merge branch 'FitnessUI' into Fitness

NoahE321 committed:
Cleaning up

MaherHssn committed:
used getClass() instead :)

MaherHssn committed:
Merge branch 'Fitness' of https://github.com/Ye-ICS/fss-everything-app into Fitness

MaherHssn committed:
Update DietMenuController.java

MaherHssn committed:
check pref now accounts for isActive variable

MaherHssn committed:
lil changes

renamed main menu controller, removed throws

B1gCD committed:
minor tweaks

B1gCD committed:
Merge branch 'FitnessMethods2' into Fitness

NoahE321 committed:
Update SleepTimer.java

MrYe070 reviewed (changes requested):
See comments.
Also, does every class need to be public?
Also also, please, more descriptive commit messages. "lil changes", "minor tweaks", etc, are quite uninformative.

NoahE321 committed:
Merge branch 'finess-methods-1' into Fitness

MaherHssn commented on app/src/main/java/com/fss/everythingapp/fitness/GeneralInfo.java:
```diff
@@ -13,15 +13,8 @@ public class GeneralInfo {
     private boolean acceptableFat;
     static private boolean obesityFat;
     public static int bodyFat;
+    public static Boolean isActive; //true means person is active, false is opposite
```
I used it in MainMenuSignedInController to check if the user selected a button by checking if the boolean was null or not

MaherHssn committed:
used .equals and specified few things

instead of using == for strings, used .equals and specified what bmr is, changed bmr method name to -> bmrCalculation

MrYe070 reviewed (changes requested):
Apart from my comments, there's also a merge conflict. Fetch and merge from main.

NoahE321 committed:
Merge branch 'main' into Fitness

MaherHssn committed:
Renamed FXML's and a few variables

changed bmr -> basalMetabloicRate
changed isActive -> isPhysicallyActive
changed isPhysicallyActive to non-nullable boolean

MaherHssn committed:
Merge branch 'Fitness' of https://github.com/Ye-ICS/fss-everything-app into Fitness

NoahE321 committed:
Added a controller, sleep menu, and updated sleeptimer

NoahE321 committed:
Merge branch 'Fitness' into finess-methods-1

MaherHssn committed:
renamed bmrCalculation method and adjusted for muscular build

MaherHssn committed:
Merge branch 'Fitness' of https://github.com/Ye-ICS/fss-everything-app into Fitness

NoahE321 committed:
Update SleepMenuController.java

NoahE321 committed:
Updating SleepMenuController and etc. to continuously update the date

MaherHssn committed:
added excersize goals to workout menu

MaherHssn committed:
added excersize bars to Workout Menu

also changed excercise to double instead of int

MaherHssn committed:
changed some stuff to doubles instead of ints

NoahE321 committed:
Updated the code for sleep classes and added a text file

NoahE321 committed:
Merge branch 'finess-methods-1' into Fitness

NoahE321 committed:
Update GeneralInfo.java

MaherHssn committed:
Merge branch 'main' into Fitness

MaherHssn committed:
post merge

renamed mainmenu.fxml to mainmenufitness in order to not conflict with the new main menu

MaherHssn committed:
references to MainMenu now correctly reference MainMenuFitness instead

NoahE321 committed:
changed text file and added code to write to file

NoahE321 committed:
Merge branch 'finess-methods-1' into Fitness

NoahE321 committed:
Updated code for file reading and writing

NoahE321 committed:
Merge branch 'finess-methods-1' into Fitness

MaherHssn committed:
Moved fxmls and userinfo files to own seperate file in resources

MaherHssn committed:
Merge branch 'Fitness' of https://github.com/Ye-ICS/fss-everything-app into Fitness

NoahE321 committed:
Merge branch 'Fitness' of https://github.com/Ye-ICS/fss-everything-app into Fitness

NoahE321 committed:
Update GeneralInfo.java

MaherHssn committed:
fixed MainMenu.fxml

NoahE321 committed:
Update FitnessMainMenuController.java

NoahE321 committed:
Merge branch 'main' into Fitness

MrYe070 committed:
Merge branch 'main' into Fitness
